dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImpl.java

```diff
@@ -1707,6 +1707,15 @@ public void deleteAllVersionsandBackup(List<Contentlet> contentlets, User user,
         }
         deleteBinaryFiles(contentletsVersion,null);
 
+        try {
+            String bla = null;
+            bla = "fffff";
+            bla = "fffff";
+            bla = "fffff";
+            bla = "fffff";
+        }catch (Exception e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1166)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either remove or fill this block of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00108)